tests/snappi/pfcwd/files/pfcwd_multidut_burst_storm_helper.py

```diff
@@ -0,0 +1,309 @@
+import time
```

Comment:
Please update the description section of the PR with details on what this PR is intending to do. Same comment for all the other PRs that have been raised in a similar format